.github/workflows/release.yml

```diff
@@ -41,7 +46,6 @@ jobs:
       - name: Publish a Python distribution to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
```

Comment:
`password` can be removed because Copulas has been added as a trusted publisher for Test PyPI.
- https://docs.pypi.org/trusted-publishers/using-a-publisher/

Note: For real PyPI, this is still a WIP.